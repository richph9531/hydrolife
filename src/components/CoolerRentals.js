import React from 'react';

import classes from '../styles/CoolerRentals.module.css';

const CoolerRentals = () => (
  <div className={classes.CoolerRentals}>
  <h1>Cooler Rentals</h1>
</div>
);

export default CoolerRentals;
