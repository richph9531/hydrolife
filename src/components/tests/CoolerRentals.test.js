import React from 'react';

import { render } from '@testing-library/react';
import CoolerRentals from '../CoolerRentals';

test('Cooler Rentals renders correctly', () => {
  const view = render(<CoolerRentals />);
  expect(view.baseElement).toMatchSnapshot();
});
