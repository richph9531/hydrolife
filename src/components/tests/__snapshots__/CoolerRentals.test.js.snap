// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cooler Rentals renders correctly 1`] = `
<body>
  <div>
    <div
      class="CoolerRentals"
    >
      <h1>
        Cooler Rentals
      </h1>
    </div>
  </div>
</body>
`;
